File: Unidad 5/JAVA/P10_U5.java
import java.util.Scanner;
public class P10_U5
{
	public static void main(String[] args) {
        int longitud = 8;
        String contrasena = aleatorio(longitud); 
        System.out.println("pasword: ,"+contrasena);
    }
    public static String aleatorio(int longitud){
        String caracteres = "ABCDEFghijkl0123456789*-_.";
        Random random = new Random();
        StringBuilder cadena = new StringBuilder(longitud);
        for (int i = 0; i<longitud; i++){
           
            int indiceCaracter = random.nextInt(caracteres.length());
            cadena.append(caracteres.charAt(indiceCaracter));
        }           
    return cadena.toString();
    }
    
}

File: Unidad 5/C#/P10_U5.cs
using System;

class P10_U5
{
    static void Main()
    {
            int longitud = 8;
            string contrasena = GenerarContrasena(longitud);
            Console.WriteLine("Contraseña: " + contrasena);
        }

        static string GenerarContrasena(int longitud)
        {
            string caracteres = "ABCDEFghijkl0123456789*-_.";
            Random random = new Random();
            char[] cadena = new char[longitud];

            for (int i = 0; i < longitud; i++)
            {
                int indiceCaracter = random.Next(caracteres.Length);
                cadena[i] = caracteres[indiceCaracter];
            }

            return new string(cadena);
        }
    }
}